0019-glossary/0019-glossary.md

```diff
@@ -30,6 +30,9 @@ When part of an account's balance is not readily usable, it is up to the ledger
 ## Centralized Ledger
 A ledger which is operated by a single entity.
 
+## Chunked Payment
+A large payment which is split into smaller chunks that are then sent sequentially. Chunked payments reduce the total amount of money in-flight at a given time, which can reduce certain risks and costs.
```

Comment:
A large application-level payment which is split into smaller interledger-level chunks that are then sent sequentially. Chunked payments reduce the total amount of money in-flight at a given time, which can reduce certain risks and costs for intermediate connectors, while increasing certain risks for the sender and receiver.